rest/route/annotations.go

```diff
@@ -319,3 +320,107 @@ func (h *annotationByTaskPutHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 	return responder
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// PUT /rest/v2/tasks/{task_id}/created_ticket
+
+// this api will be used by teams who set up their own web hook to file a ticket, to send
+// us the information for the ticket that was created so that we can store and display it.
+
+type createdTicketByTaskPutHandler struct {
+	taskId    string
+	execution *int
+	user      gimlet.User
+	ticket    *model.APIIssueLink
+	sc        data.Connector
+}
+
+func makeCreatedTicketByTask(sc data.Connector) gimlet.RouteHandler {
+	return &createdTicketByTaskPutHandler{
+		sc: sc,
+	}
+}
+
+func (h *createdTicketByTaskPutHandler) Factory() gimlet.RouteHandler {
+	return &createdTicketByTaskPutHandler{
+		sc: h.sc,
+	}
+}
+
+func (h *createdTicketByTaskPutHandler) Parse(ctx context.Context, r *http.Request) error {
+	var err error
+	h.taskId = gimlet.GetVars(r)["task_id"]
+	if h.taskId == "" {
+		return gimlet.ErrorResponse{
+			Message:    "task ID cannot be empty",
+			StatusCode: http.StatusBadRequest,
+		}
+	}
+	// for now, tickets will be created for a specific task execution only
+	// and will not be visible on other executions
+	if r.URL.Query().Get("execution") == "" {
+		return gimlet.ErrorResponse{
+			Message:    fmt.Sprintf("the task execution must be specified"),
+			StatusCode: http.StatusBadRequest,
+		}
+	}
+	execution, err := strconv.Atoi(r.URL.Query().Get("execution"))
```

Comment:
Rather than call r.URL.Query().Get("execution" twice, can you just assign this to a variable before 362? Also, you should check this error